### PR TITLE
docs: Added app changelog

### PR DESCRIPTION
**Pull request by machour**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+* Added German translations (#458)
+* Added Galician translations (#452)
+* Fixed animated gif display on Android (#446)
+* Added portuguese translations (#440)
+* Fixed README file display on Android (#427)
+* Added check for PR mergeable status and disabled button accordingly (#425)
+* Added "Open in browser" functionality in various screens (#424)
+* Added spanish translations (#282)
+
+## Version 1.3.0 (2017-10-04)
```
Comment (lex111) [inline CHANGELOG.md]: Will there be no releases of bug fixes? Version 1.3.1 is missing.

Comment (machour) [reply CHANGELOG.md]: It's not out yet as far as I understand

Comment (lex111) [reply CHANGELOG.md]: image

Comment (machour) [reply CHANGELOG.md]: yes, but it wasn't released yet. @housseindjirdeh mentioned on gitter that he'll be releasing it yesterday. Not sure if it was done, and what was included.

Comment (lex111) [reply CHANGELOG.md]: No, this release with bug fixes has already been released (check Google Play), @housseindjirdeh meant a new release - 1.3.2.

Comment (lex111) [reply CHANGELOG.md]: ... but he did not update `package.json`, I wrote about it, but the release took place.

Comment (machour) [reply CHANGELOG.md]: Fixed, thanks a lot for the clarifications!

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+* Fixed issues links parsing (#463)
+* Added German translations (#458)
+* Added Galician translations (#452)
+* Fixed animated gif display on Android (#446)
+* Added portuguese translations (#440)
+* Added check for PR mergeable status and disabled button accordingly (#425)
+* Added "Open in browser" functionality in various screens (#424)
+* Added spanish translations (#282)
+
+## Version 1.3.1 (2017-10-04)
```
Comment (andrewda) [inline CHANGELOG.md]: Pretty minor, but maybe link these versions to the release on GitHub?

**Comment by andrewda** [comment]
Also maybe we could eventually show this in the app near where we have the Privacy Agreement and that sort of stuff.

**Comment by housseindjirdeh** [comment]
<3 Thank you @machour

**Comment by machour** [comment]
@andrewda Added releases link.

Let's move this conversation to #505 where @lex111 proposed a lib to generate this file automatically

Mehdi

**Comment by machour** [comment]
Closed in favor of #696

**Pull request closed without merging**